events/events.go

```diff
@@ -478,6 +483,22 @@ type AccountStatusUpdatedData struct {
 	NewState string `json:"new_state"`
 }
 
+// AccountUpdated represents an account update by an actor other than the user
+type AccountUpdated struct {
+	BaseBody
+	Data *AccountUpdatedData `json:"data"`
+}
+
+// AccountUpdatedData holds the event specific data
+type AccountUpdatedData struct {
+	Reason   string  `json:"reason"`
```

Comment:
Does `json:"reason"` mean that when this struct is serialized to JSON, the key will be named `reason`?

Reply:
Yup!